_locales/_untranslated_en.json

```diff
@@ -23,5 +23,41 @@
   },
   "pref_current_account": {
     "message": "Current Account"
+  },
+  "onboarding_create_instant_account": {
+    "message": "Let's get started!"
+  },
+  "onboarding_alternative_logins": {
+    "message": "I already have a login"
+  },
+  "manual_account_setup_option": {
+    "message": "Manual login"
+  },
+  "instant_onboarding_title": {
+    "message": "Your profile"
+  },
+  "instant_onboarding_create": {
+    "message": "Create profile"
+  },
+  "instant_onboarding_default_info": {
+    "message": "Your profile will be created on the default instance."
```

Comment:
if some additional policy is needed at the end, we'll probably add/link that below this text - but that seems to be an open question

Reply:
this string will be removed completely, instead a "agree to privacy policy" link will be added above the "create button"